app/views/samples/_form.html.erb

```diff
@@ -25,5 +25,20 @@
   </div>
 
   <%= form_submit_buttons(@sample, validate:false) %>
+  <script type="text/javascript">
+  // Get the value of all tags inputs of type sample_multi as an array and set their sibling hidden input values
+   const tagsInputMulti = $j("#sample_attributes input[data-role='seek-objectsinput']")
+    $j("form[id*='sample']").on("submit", function(e){
+        $j.each(tagsInputMulti, (i,item) => {
+          const valuesArray = $j(item).val().split(",");
+          $j.each(valuesArray, (j,value) => {
+            $j(this).append(`<input type='hidden' name='${$j(item).attr('name')}[]' value='${value}'>`)
+          })
+          $j(item).siblings('input:hidden:first').val(JSON.stringify(valuesArray))
+          $j(item).attr('name', '')
+        })
+      });
+     
+  </script>
```

Comment:
Is this needed?

The `objects_input` in the form should provide the params as a comma separated list of IDs, that get handled by the `convert` method in the attribute type handler.